supertokens_python/recipe/emailpassword/syncio/__init__.py

```diff
@@ -79,12 +79,17 @@ def sign_in(email: str, password: str, user_context: Union[None, Dict[str, Any]]
     return sync(sign_in(email, password, user_context))
 
 
-async def sign_up(email: str, password: str, user_context: Union[None, Dict[str, Any]] = None):
```

Comment:
This fix should go in another PR to the main x.y branch as it is unrelated to this PR.

Reply:
Fixed in https://github.com/supertokens/supertokens-python/pull/168/files